pkg/common/utils.go

```diff
@@ -30,6 +30,14 @@ import (
 
 	"github.com/apache/yunikorn-core/pkg/log"
 	interfaceCommon "github.com/apache/yunikorn-scheduler-interface/lib/go/common"
+	"github.com/apache/yunikorn-scheduler-interface/lib/go/si"
+)
+
+const CreationTime = "CreationTime"
```

Comment:
This should be moved to the SI as it used in both shim and core. Needs to be added to YUNIKORN-1122 like you did with `ready`.